### PR TITLE
fix: spelling error

### PR DESCRIPTION
**Pull request by saifeiLee**
## Description

FIx some spelling error.

## Motivation and Context

Word spelling errors make people confuse.

## How Has This Been Tested?

`npm run test` result passed.

## Types of changes

- [x ] Updated docs / Refactor code / Added a tests case (non-breaking change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I have updated the changelog.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -44,6 +44,18 @@
...
+      "name": "Debug Current TS file",
+      "type": "node",
+      "request": "launch",
+      "runtimeExecutable": "node",
+      "runtimeArgs": [
+        "--nolazy", "-r", "ts-node/register/transpile-only"
+      ],
+      "program": "${file}",
+      "cwd": "${workspaceFolder}",
+      "internalConsoleOptions": "openOnSessionStart",
```
Comment (notaphplover) [inline .vscode/launch.json]: This is out of the scope of this PR

File: src/planning/planner.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import * as ERROR_MSGS from '../constants/error_msgs';
 import { BindingTypeEnum, TargetTypeEnum } from '../constants/literal_types';
 import * as METADATA_KEY from '../constants/metadata_keys';
 import { interfaces } from '../interfaces/interfaces';
-import { isStackOverflowExeption } from '../utils/exceptions';
+import { isStackOverflowException } from '../utils/exceptions';
```
Comment (notaphplover) [inline src/planning/planner.ts:6]: :heart:

**Review by notaphplover — COMMENTED** [review]
Nice, let's remove the script and merge it :)

File: .vscode/launch.json (hunk 1 above)
Comment (saifeiLee) [reply .vscode/launch.json]: > This is out of the scope of this PR

Should I revert this commit and update this PR?

Comment (notaphplover) [reply .vscode/launch.json]: Or simply add a commit removing the changes at `launch.json`.

Comment (saifeiLee) [reply .vscode/launch.json]: Done. :)

**Review by notaphplover — APPROVED** [review]

**Pull request merged**